static/src/stylesheets/layout/new-header/_new-header.scss

```diff
@@ -98,57 +97,73 @@ from scrolling */
     }
 }
 
+.new-header__menu-toggle {
+    &:focus {
+        outline: 0;
```

Comment:
Is there a proper `::focus` style replacement, which is accessible?

Reply:
Will sort this out now.